src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFDocument.java

```diff
@@ -1603,9 +1603,69 @@ public XWPFTableCell getTableCell(CTTc cell) {
         }
         return tableRow.getTableCell(cell);
     }
-
+    
     @Override
     public XWPFDocument getXWPFDocument() {
         return this;
     }
+    
+    /**
+     * This method is used to create template for chart XML
+     * no need to read MS-Word file and modify charts
+     * @return This method return object of XWPFChart Object with default height and width
+     * @throws InvalidFormatException
+     * @throws IOException
+     * @since POI 4.0.0
+     */
+    public XWPFChart addChart() throws InvalidFormatException, IOException{
+        return addChart(XWPFChart.DEFAULT_WIDTH, XWPFChart.DEFAULT_HEIGHT);
+    }
+    
+    /**
+     * This method is used to create template for chart XML
+     * no need to read MS-Word file and modify charts
+     * @param width width of chart
+     * @param height height of chart
+     * @return This method return object of XWPFChart
+     * @throws InvalidFormatException
+     * @throws IOException
+     * @since POI 4.0.0
+     */
+    public XWPFChart addChart(int width, int height) throws InvalidFormatException, IOException{
+        
+        //get chart number
+        int chartNumber = getPackagePart().getPackage().
+                getPartsByContentType(XWPFRelation.CHART.getContentType()).size() + 1;
+        
+        //create relationship in document for new chart
+        RelationPart rp = createRelationship(
+                XWPFRelation.CHART, XWPFFactory.getInstance(), chartNumber, false);
+        
+        //get chart relationship id
+        String chartId = rp.getRelationship().getId();
+        
+        //create paragraph and run object
+        XWPFRun xRun = this.createParagraph().createRun();
+        
+        CTInline inline = xRun.addChart(width, height,chartId);
```

Comment:
No need to pass `chartId` here, if you place the initialization (from `XWPFRun`, see comment there) in a private method `appendChartElement` in this class, where you will pass `inline` and `chartId` as arguments.